### PR TITLE
Create Bad Response Code : 404.md

### PR DESCRIPTION
**Pull request by mcseagate**

### CODE REVIEW

File: Bad Response Code : 404.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Pangram_API_Test
+
+Issue: APi Get request returns a 404 response code which defaults to A Bad Request
+YOu are required to provide a valid API URI for a successful completion of the project.
```
Comment (mlaguren) [inline Bad Response Code : 404.md]: Per the swagger docs, there is no GET request for this API

**Comment by mcseagate** [comment]
There's another post request there. Please pull the most updated code

**Comment by mcseagate** [comment]
Look closely, there is a POST request there. I deliberately did a POST and a GET request for this assignment. You may have to PULL the latest code. Thanks

**Comment by melvin-mbww** [comment]
I have pulled the latest code, but there have been no new commits on either branch for 2 days.

**Comment by mcseagate** [comment]
I merged again and sent you another invite to a different branch. You should see a PostTest.java class. That's where the POST request exists

**Comment by melvin-mbww** [comment]
Will review, not sure why you had to create a second repository for this.  I see the post test in the new repo, still not sure why you don't have it here as you mentioned earlier.

**Comment by mcseagate** [comment]
I figured I merged to Master but gave you access to a different branch.
That was the confusion. This was why you have a new invite to contribute to
Master. The error is regrettable.

**Comment by melvin-mbww** [comment]
You have two repositories that you have invited me to:


https://github.com/mcseagate/Pangram_API_Test which is the initial repository with the pull request.  If you perform a search via Github or locally on both master and the branch, the post test you refer to is not found.


https://github.com/mcseagate/Pangram_API_Test_Update_2A is an updated version of the repository.  In this repository, I see the post test.  This is my primary question as to why you did not just update the original branch.


Anyway, both repos are being reviewed and feedback will be sent back to Holly.  Thank you again for taking the time to discuss with me on the position.



________________________________

**Comment by mcseagate** [comment]
Please accept the one that says Pangram-API_Test_Update_2A. That is pointed
to Master where I have the merged code. Thanks.

**Pull request closed without merging**